WORKSPACE

```diff
@@ -142,37 +142,19 @@ http_file(
 
 http_file(
     name = "fedora_image",
-    sha256 = "a30549d620bf6bf41d30a9a58626e59dfa70bb011fd7d50f6c4511ad2e479a39",
+    sha256 = "3ff094bc37d6ddf21c617e479a84597b2c15e06003e8f5b4c3a0529979626ef9",
     urls = [
-        "https://download.fedoraproject.org/pub/fedora/linux/releases/29/Cloud/x86_64/images/Fedora-Cloud-Base-29-1.2.x86_64.qcow2",
-        "https://storage.googleapis.com/builddeps/a30549d620bf6bf41d30a9a58626e59dfa70bb011fd7d50f6c4511ad2e479a39",
+        "https://download.fedoraproject.org/pub/fedora/linux/releases/test/32_Beta/Cloud/x86_64/images/Fedora-Cloud-Base-32_Beta-1.2.x86_64.qcow2",
```

Comment:
Let's use the released Fedora32 image instead of beta
https://download.fedoraproject.org/pub/fedora/linux/releases/32/Cloud/x86_64/images/Fedora-Cloud-Base-32-
1.6.x86_64.qcow2

Reply:
Done.